src/DependencyInjection/Providers/WebflowProviderConfigurator.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/*
+ * OAuth2 Client Bundle
+ * Copyright (c) KnpUniversity <http://knpuniversity.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace KnpU\OAuth2ClientBundle\DependencyInjection\Providers;
+
+use KnpU\OAuth2ClientBundle\Client\Provider\WebflowClient;
+use Symfony\Component\Config\Definition\Builder\NodeBuilder;
+
+class WebflowProviderConfigurator implements ProviderConfiguratorInterface
+{
+    public function buildConfiguration(NodeBuilder $node)
+    {
+        // no custom options
+    }
+
+    public function getProviderClass(array $config)
+    {
+        return 'Koalati\OAuth2\Client\Provider\Webflow';
```

Comment:
We can use `return Webflow::class` here too instead of hardcoding this as a string.. but don't forget to use the namespace too :)

Reply:
Done! :)